<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerInterface</class>
 <widget class="QMainWindow" name="ServerInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/* Центральный виджет (centralwidget) */
        QWidget#centralwidget {
            border: 2px solid #cccccc; /* Рамка */
            border-radius: 10px; /* Закругление углов */
			margin: 5px;
        }

        /* Виджеты внутри главного окна (QWidget) */
        QWidget {
            background-color: #FAF9F9;
        }

		#widget {
			border: 1px solid #cccccc; /* Рамка */
		}

        /* Панель меню (menubar) */
        QMenuBar#menubar {
            background-color: #FAF9F9; /* Цвет фона */
            padding: 5px 10px; /* Отступы */
        }

        /* Заголовки вкладок (QTabBar::tab) */
        QTabBar::tab {
            background-color: #f0f0f0; /* Цвет фона */
            color: #333333; /* Цвет текста */
            padding: 10px 20px; /* Отступы */
            border-top-left-radius: 10px; /* Закругление углов */
            border-top-right-radius: 10px;
            border-bottom: 1px solid transparent; /* Убираем нижнюю границу */
        }
		/* Отступ справа для QTabWidget */

#tabWidget {
 margin: 10px; /* Сокращаем размер виджета вкладок на 10px справа */
}
        /* Выбранная вкладка (QTabBar::tab:selected) */
        QTabBar::tab:selected {
            background-color: #007bff; /* Цвет фона */
            color: #ffffff; /* Цвет текста */
        }

        /* Комбо-бокс (QComboBox) */
        QComboBox {
            padding: 8px; /* Отступы */
            background-color: #ffffff; /* Цвет фона */
            border: 1px solid #cccccc; /* Рамка */
            border-radius: 5px; /* Закругление углов */
            min-width: 150px; /* Минимальная ширина */
            font-size: 14px; /* Размер шрифта */
            color: #333333; /* Цвет текста */
        }
QComboBox QAbstractItemView {
    /* Установите фоновый цвет */
    border: 1px solid #cccccc;
	border-radius: 5px;
}

QComboBox::item {
    /* Установите высоту элементов */
    height: 30px;
}

QComboBox::drop-down {
    width: 16px; /* Ширина стрелки */
    height: 16px; /* Высота стрелки */
	background-color: yellow;
	background-image: url(qrc:/arrow.png);
}

        /* Кнопка (QPushButton) */
        QPushButton {
            padding: 8px 16px; /* Отступы */
            background-color: #007bff; /* Цвет фона */
            color: #ffffff; /* Цвет текста */
            border: none; /* Убираем границы */
            border-radius: 5px; /* Закругление углов */
            font-size: 14px; /* Размер шрифта */
        }

        /* Наведение на кнопку (QPushButton:hover) */
        QPushButton:hover {
            background-color: #0056b3; /* Цвет фона при наведении */
        }

        /* Чекбокс (QCheckBox) */
        QCheckBox {
            spacing: 5px; /* Расстояние между флажком и текстом */
            font-size: 14px; /* Размер шрифта */
            color: #333333; /* Цвет текста */
        }

        /* Текстовое поле (QTextBrowser) */
        QTextBrowser {
            background-color: #ffffff; /* Цвет фона */
            border: 1px solid #cccccc; /* Рамка */
            border-radius: 5px; /* Закругление углов */
            padding: 10px; /* Отступы */
            min-height: 150px; /* Минимальная высота */
            font-size: 14px; /* Размер шрифта */
            color: #333333; /* Цвет текста */
        }
		QTableView {
			background-color: #ffffff; /* Цвет фона */
            border: 1px solid #cccccc; /* Рамка */
            font-size: 14px; /* Размер шрифта */
            color: #333333; /* Цвет текста */
			margin: 0px 0px 0px 10px;
		}
		QTreeView {
			background-color: #ffffff; /* Цвет фона */
            border: 1px solid #cccccc; /* Рамка */
            border-radius: 5px; /* Закругление углов */
            padding: 10px; /* Отступы */
            font-size: 14px; /* Размер шрифта */
            color: #333333; /* Цвет текста */
		}
        /* Кнопка инструментов (QToolButton) */
        QToolButton {
            padding: 8px 16px; /* Отступы */
            background-color: #007bff; /* Цвет фона */
            color: #ffffff; /* Цвет текста */
            border: none; /* Убираем границы */
            border-radius: 5px; /* Закругление углов */
            font-size: 14px; /* Размер шрифта */
        }

        /* Наведение на кнопку инструментов (QToolButton:hover) */
        QToolButton:hover {
            background-color: #0056b3; /* Цвет фона при наведении */
        }
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QWidget" name="tableProperties" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>3</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QToolButton" name="filterButton">
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">#filterButton {
	padding: 0;
}
#filterButton::menu-indicator { image: none; }</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/buttons/icon/filter.svg</normaloff>:/buttons/icon/filter.svg</iconset>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="search" native="true">
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">#search {
	background-color: white;
	border: 2px solid #959595;
	border-radius: 10px;
	padding: 10px;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QLineEdit" name="searchLine">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>3</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">#searchLine {
	border: none;
	background-color: transparent;
	margin: 0;
	padding:0;
}

</string>
            </property>
            <property name="placeholderText">
             <string>Search...</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="searchButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">#searchButton {
	background-color: transparent;
	color: black;
	margin: 0;
	padding: 0;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/buttons/icon/search.svg</normaloff>:/buttons/icon/search.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QWidget" name="appliedFilters" native="true">
         <property name="styleSheet">
          <string notr="true">#appliedFilters &gt; * {
    border: 2px solid #959595;
    border-radius: 10px;
    padding: 0 10px;
    background-color: #ffffff;
}
#appliedFilters QWidget QPushButton {
	margin: 0;
	padding: 0;
	background-color: transparent;
}
#appliedFilters QWidget QPushButton::hover {
	background-color: rgba(55, 55, 55, 50);
	border-radius: 10px;
	border: 2px solid #888888;
}
#appliedFilters QWidget QLabel {
	color: #666666;
	padding: 0px;
	margin: 0px;
	font-weight: 700;
	background-color: transparent;
}
</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" rowspan="3">
     <widget class="QTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>3</horstretch>
        <verstretch>17</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QWidget" name="widget_3" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>8</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="markSetManual">
         <property name="text">
          <string>Manual</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="markCloseDrug">
         <property name="text">
          <string>Close DRUG.EXE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="serverUpdateButton">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="serverArchiveButton">
         <property name="text">
          <string>Archive</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="serverClearButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="2" rowspan="2">
     <widget class="QTreeView" name="errorTreeView"/>
    </item>
    <item row="0" column="2" rowspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>5</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>8</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <widget class="QWidget" name="tab">
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <attribute name="title">
        <string>MAIN</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTextBrowser" name="textBrowser_2">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:14px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <attribute name="title">
        <string>DETAILS</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:14px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
